### ISSUE TITLE
Multiple monitors

### ISSUE THREAD

**Issue by QualiMock** [fawtytoo/Show-Application-View-When-Workspace-Empty#1]
Hello! I don't get to the applications view with applications on the second monitor

**Comment by fawtytoo** [comment]
That's correct. If one of the monitors shows an open window then the workspace is not empty.
